src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRun/LocalMethodImport.cs

```diff
@@ -53,5 +53,13 @@ public override IEnumerable<DependencyListEntry> GetStaticDependencies(NodeFacto
             }
             yield return new DependencyListEntry(_localMethod, "Local method import");
         }
+
+        public override int CompareToImpl(ISortableNode other, CompilerComparer comparer)
+        {
+            int result = comparer.Compare(_localMethod, ((LocalMethodImport)other)._localMethod);
+            if (result != 0) return result;
+
+            return base.CompareToImpl(other, comparer);
```

Comment:
Missing _signatureContext in the comparison

Reply:
Should be compared if/when the MethodFixupSignature's comparison function is called.

Reply:
ok so let's delete the field then to reduce confusion.